src/Components/test/E2ETest/ServerExecutionTests/CircuitGracefulTerminationTests.cs

```diff
@@ -40,7 +40,7 @@ protected override void InitializeAsyncCore()
     {
         Navigate(ServerPathBase);
         Browser.MountTestComponent<GracefulTermination>();
-        Browser.Equal("Graceful Termination", () => Browser.Exists(By.TagName("h1")).Text);
+        Browser.Equal("Graceful Termination", () => Browser.Exists(By.CssSelector("app h1")).Text);
```

Comment:
This is still the same element, we just select it more precisely, not to confuse it with the h1 that is not embedded in `<app>`.